### ISSUE TITLE
Error in 3-axis gyroscope frequency? (3200 Hertz...)

### ISSUE THREAD

**Issue by GSORF** [gopro/gpmf-parser#35]
Hi,

I love this feature of having detailed telemetry data with newer GoPro cameras, thank you very much at all the People involved in this feature.
Furthermore, this documentation is already great, but could always use a bit more love. I am currently trying to retrieve the data for sensor data fusion and will contribute any time I find something.

And I think I already stumbled across one thing in the readme.md:
At the Bottom of the page the Details for the GoPro Fusion (see https://github.com/gopro/gpmf-parser#fusion-adds-and-changes) read:

> GYRO | 3-axis gyroscope | 3200 | rad/s | Data order -Y,X,Z

Just wanted to check: Does the Fusion really have a measurement frequency of 3200 Hz for its gyroscope? Could this be a typo? This looks suspicious as all other sampling frequencies are at 200 Hz.

Thanks for any insight into that. I am also happy to create a PR in case you confirm this issue.

**Comment by dnewman-gpsw** [comment]
Yes, Fusion has GYRO at around 3200Hz.  The high frequency is used for sub-frame rolling shutter reduction and ultra smooth stabilization.
